### DISCUSSION TITLE
get_friend_list failed

### DISCUSSION THREAD

**Discussion by wakeshu** [howmanybots/onebot#143] [问答]
get_friend_list failed ！！！

**Comment by ExerciseBook** [comment]
您好，onebot 仅为一个协议的规范，不能够知晓您所使用的 onebot 实现是否成功实现了这部分的功能。建议您到您所使用的插件的 repo 下进行询问。

如果您不知道您使用的是哪一个插件，您可以将其的基本信息描述出来让热心网友一同帮您解决。
